components/TextFields/src/ContainedInputView/MDCBaseTextField.h

```diff
@@ -55,4 +55,58 @@
  */
 @property(nonatomic, assign) UITextFieldViewMode trailingViewMode;
 
+/**
+ Sets the floating label color for a given state.
+ Floating label color refers to the color of the label when it's in its "floating position," i.e.
+ when it's above the text area.
+ @param floatingLabelColor The UIColor for the given state.
+ @param state The UIControlState. The accepted values are UIControlStateNormal,
+ UIControlStateDisabled, and MDCTextControlStateEditing, which is a custom MDC
+ UIControlState value.
+ */
+- (void)setFloatingLabelColor:(nonnull UIColor *)floatingLabelColor forState:(UIControlState)state;
```

Comment:
In UIButton these colors are nullable

https://developer.apple.com/documentation/uikit/uibutton/1623993-settitlecolor?language=objc

We spoke in person about this and I dont want to forget to chat about this more.